platform/migration/platform-migratable-util/pom.xml

```diff
@@ -90,18 +90,18 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.9</minimum>
```

Comment:
❓ Since commenting out the test cases is temporary, should we leave the original ratios in comments?